tests/unit/date-utilities.spec.ts

```diff
@@ -0,0 +1,190 @@
+import { DateUtilities } from '@/services'
```

Comment:
I created this test file for the `date-utilities`, then copied over the tests from `date-mixins.spec.ts` and modified them a bit to get them to work. The `date-mixins` are deprecated but are still used in a few places in the application.

At the bottom I added new tests for `addMonthsToDate` and `subtractDates`